laws/src/main/scala/cats/laws/discipline/NonEmptyTraverseTests.scala

```diff
@@ -44,13 +44,11 @@ trait NonEmptyTraverseTests[F[_]] extends TraverseTests[F] with ReducibleTests[F
     ArbYB: Arbitrary[Y[B]],
     ArbYC: Arbitrary[Y[C]],
     ArbFB: Arbitrary[F[B]],
-    ArbFM: Arbitrary[F[M]],
     ArbXM: Arbitrary[X[M]],
     ArbYM: Arbitrary[Y[M]],
-    ArbFGA: Arbitrary[F[G[A]]],
+    ArbGU: Arbitrary[G[Unit]],
+    ArbFGU: Arbitrary[F[G[Unit]]],
     ArbFXM: Arbitrary[F[X[M]]],
-    ArbGB: Arbitrary[G[B]],
-    ArbGM: Arbitrary[G[M]],
```

Comment:
These changes will also cause compatibility problems.

Reply:
I assumed the correct answer here would be to leave the prior implicit params alone and just add mine at the end of arglist. the mima check is still complaining in 2.12 and I'm not sure how to resolve it

Reply:
Unfortunately we cannot change these signatures at all: no changes, no additions, no removals. It's not specific to Scala 2.12, that one probably just happened to fail first.

See https://github.com/typelevel/cats/pull/4324#discussion_r1007423936 for a possible strategy.